controllers/ListItemController.php

```diff
@@ -31,6 +61,45 @@ public function get_list_items(string $listID)
 			echo json_encode([ "success" => true, "data" => $result ]);
 		}
 
+		/**
+			* Swagger doc block
+			*
+			* @OA/Post(
+			* path="list/item/{listID}",
+			* summary = "Add a new list item",
+			* @OA\RequestBody(
+     	* 	required=true,
+			* 	@OA\Property(
+			*  		description = "List item to be added",
+			*  		name = "name",
+			*  		type = "string",
+			*  		in = "body",
+			*  		required = true,
+			*  		minLength = 3,
+			*  		maxLength = 250
+			* 	),
+			* 	@OA\Property(
+			*  		description = "Is the item repeating",
+			*  		name = "is_repeating",
+			*  		type = "integer",
+			*  		in = "body",
+			*  		required = true
+			* 	),
+			* 	@OA\Property(
+			*  		description = "Image file to upload for the list item",
+			*  		name = "name",
+			*  		type = "string",
+			*			format = "file",
+			*  		in = "body",
+			*  		required=false
+			* 	),
+			* ),
+			*	@OA\Response(
+			* 	response="401",
+			*   description="Invalid token",
+		  * ),

```

Comment:
Here it is missing a description of what the error response body looks like 
 which can be done using this property `@OA\JsonContent(ref="#/components/schemas/error_response")`

Where the `error_response` is a new `@OA\Schema` that matches the output of the `error_response` function from the  [utility.php](https://github.com/gigili/did-you-buy-it-api/blob/32b4aec126d5c0cbbd5753abbb30dd947a203490/utility/helpers.php#L59) file